### DISCUSSION TITLE
some of my clients getting this view:

### DISCUSSION THREAD

**Discussion by grfityhero** [add2cal/add-to-calendar-button#635] [Q&A]
some of my clients getting this view in their mobile devices (looks like a desktop view)
![image](https://github.com/user-attachments/assets/d9765cad-10d8-421c-9023-7c103525973b)

**Comment by jekuer** [comment]
Hi @grfityhero ,

To be able to debug this:
- What do you mean with "clients"? Users of your website or people you build websites for? Basically, question is: who is master over the button code, who over the input data?
- What version of the script are you using?
- Can you share the config of a button, where this is happening?
- Can you share the device and Browser used in those cases?

Thanks!
